purduesigbots/ARMS
# ⬆️ Upgrade ARMS to kernel 3.8.0

#### Overview:
Upgrade ARMS to the latest PROS kernel so that ARMS stays up to date. 

This also fixes the issue with the copy constructor changes in PROS 3.8.0 causing ARMS to crash the entire program. After some testing, it was found that simply upgrading ARMS to 3.8.0 and recompiling fixed the issue.

#### Testing:
1) Build this version of ARMS as a template
2) Create a PROS 3.8.0 project
3) Apply this version of ARMS to the project
4) Run the program with `ENCODER_TYPE`  in `config.h` set to `arms::odom::ENCODER_ADI` and one of the encoder ports set to non-zero.

The program should run like normal. If a black screen occurs, the crashing issue was not actually fixed.
